shuup/admin/modules/media/static_src/media/browser/components/listFileView.js

```diff
@@ -6,13 +6,12 @@
  * This source code is licensed under the OSL-3.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-const _ = require("lodash");
-const m = require("mithril");
-const moment = require("moment");
-const wrapFileLink = require("./wrapFileLink");
-const folderLink = require("./folderLink");
-const menuManager = require("../util/menuManager");
```

Comment:
this guy is missing
![image](https://user-images.githubusercontent.com/8770370/41131935-6d385986-6a94-11e8-90df-e0a62f4f5bca.png)